packages/quantic/force-app/main/default/lwc/quanticFacet/e2e/quanticFacet.e2e.ts

```diff
@@ -0,0 +1,209 @@
+import {testSearch, testInsight, expect} from './fixture';
+import {
+  useCaseEnum,
+  useCaseTestCases,
+} from '../../../../../../playwright/utils/useCase';
+import facetData from './data';
+
+const fixtures = {
+  search: testSearch as typeof testSearch,
+  insight: testInsight as typeof testInsight,
```

Comment:
is the `as typeof` necessary here? Would it not preserve its type otherwise?

Reply:
updated!